governance.md

```diff
@@ -0,0 +1,393 @@
+and that absence is interpreted as assent.  By a minimum of participation we
+mean that we require the participation of a certain number of individuals based
+on the importance of the issue; for non-contentious issues a single Council
+member may move forward after a suitable time, while for contentious issues we
+will often require a few, often from different institutions.
+
+The interested reader may wish to also read
+[NumPy governance section](https://docs.scipy.org/doc/numpy/dev/governance/governance.html#consensus-based-decision-making-by-the-community)
+on this topic.
+
+
+### Private communications of the Council
+
+Unless specifically required, all Council discussions and activities will be
+public and done in collaboration and discussion with the Project Contributors
+and Community. The Council will have a private mailing list that will be used
+sparingly and only when a specific matter requires privacy. When private
+communications and decisions are needed, the Council will do its best to
+summarize those to the Community after eliding personal/private/sensitive
+information that should not be posted to the public internet.
+
+### Subcommittees
+
+The Council can create subcommittees that provide leadership and guidance for
+specific aspects of the project. Subcommittees are created using the standard
+voting procedure (lazy consensus). The members of the subcommittee need
+not all be Steering Council Members, but they must have at least one Steering
+Council Member to operate. If the Steering Council Member leaves, the
+subcommittee is automatically and immediately dissolved. The subcommittee can
+also choose to self-dissolve. The Steering Council is also capable of
+dissolving the subcommittee at their discretion. Dissolution may also be part
+of the subcommittee's scope (e.g. when their task is complete).
+
+When a subcommittee is proposed, it must include an intended scope over which
+that subcommittee will focus. Within that scope the subcommittee is trusted to
+make decisions on behalf of the larger group (through the standard voting
+procedure). Though they are encouraged to check-in with the steering council to
+provide clarity on direction. For items outside of the subcommittee's scope,
+they must defer to the steering council and/or appropriate subcommittee(s) (if
+they exists).
+
+Like the Council as a whole, subcommittees should conduct their business in an
+open and public manner unless privacy is specifically called for. Private
+subcommittee communications should happen on the main private mailing list of
+the Council unless specifically called for.
+
+The BDFL retains override authority over subcommittees, but should typically
+appoint a delegate who plays that role most of the time. This delegate must
+also be a Steering Council Member.
+
+### NumFOCUS Subcommittee
+
+The Council will maintain one narrowly focused subcommittee to manage its
+interactions with NumFOCUS.
```

Comment:
You might add that this subcommittee is a requirement of fiscal sponsorship by NumFOCUS

Reply:
Sounds good.

Reply:
Thanks for the suggestion. Added some text to this effect.